backend/src/test/java/reviewme/member/repository/GithubReviewerGroupRepositoryTest.java

```diff
@@ -0,0 +1,50 @@
+package reviewme.member.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static reviewme.fixture.ReviewerGroupFixture.리뷰_그룹;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import reviewme.member.domain.GithubReviewerGroup;
+import reviewme.member.domain.Member;
+import reviewme.member.domain.ReviewerGroup;
+
+@DataJpaTest
+class GithubReviewerGroupRepositoryTest {
+
+    @Autowired
+    GithubReviewerGroupRepository githubReviewerGroupRepository;
+
+    @Autowired
+    ReviewerGroupRepository reviewerGroupRepository;
+
+    @Autowired
+    MemberRepository memberRepository;
+
+    @Test
+    void 깃허브_아이디와_리뷰어_그룹이_모두_일치하는_깃허브_리뷰어_그룹이_있는_경우를_확인한다() {
+        String githubId = "ted";
+        Member ted = new Member("테드", githubId);
+        memberRepository.save(ted);
+        ReviewerGroup reviewerGroup = 리뷰_그룹.create(ted);
+        reviewerGroupRepository.save(reviewerGroup);
+        githubReviewerGroupRepository.save(new GithubReviewerGroup(githubId, reviewerGroup));
+
+        boolean actual = githubReviewerGroupRepository.existsByGithubIdAndReviewerGroup(githubId, reviewerGroup);
+        assertThat(actual).isTrue();
+    }
```

Comment:
이 부분이 논리적으로는 맞지 않는 것 같아요..!
reviewerGroup의 fixture로 들어가는 member는 리뷰'이'여야 합니다.
지금은 리뷰'어'를 넣고 있네요😲
이를테면, 테드가 자기에 대해 리뷰해달라는 그룹에 테드를 넣은 상태같습니다!

개인적으로는 이런 실수를 방지하기 위해서 변수의 이름으로 의미를 표현하는걸 선호합니다!
예를 들어서 아래 코드 같이요

```suggestion
@Test
    void 깃허브_아이디와_리뷰어_그룹이_모두_일치하는_깃허브_리뷰어_그룹이_있는_경우를_확인한다() {
        // given
        String revieweeGithubId = "kirby";
        Member reviewee = new Member("커비", revieweeGithubId);
        String reviewerGithubId = "ted";
        Member reviewer = new Member("테드", reviewerGithubId);
        memberRepository.saveAll(List.of(reviewee, reviewer));

        ReviewerGroup reviewerGroup = 리뷰_그룹.create(reviewee);
        reviewerGroupRepository.save(reviewerGroup);
        githubReviewerGroupRepository.save(new GithubReviewerGroup(reviewerGithubId, reviewerGroup));

        // when
        boolean actual = githubReviewerGroupRepository.existsByGithubIdAndReviewerGroup(reviewerGithubId, reviewerGroup);
        
        // then
        assertThat(actual).isTrue();
    }
```

Reply:
으악 리뷰어가 리뷰어를 만드는 상황을 만들어버렸군요!😂
찾아내줘서 고마워요 산쵸! (+ 직접 수정 코드까지😋)
확실히 reviewer, reviewee라고 변수명을 지정하는 것이 덜 헷갈리겠군요!